packages/ui/nodes/media/src/ImageElement/ImageElement.types.ts

```diff
@@ -45,6 +46,8 @@ export interface ImageElementProps
 
   caption?: ImageElementPropsCaption;
 
+  popoverProps?: PopoverOptions;
```

Comment:
```suggestion
  popoverProps?: PopoverProps;
```
Then we can remove `PopoverOptions`